tests/test_client.py

```diff
@@ -22,36 +22,6 @@ def test_push_job(self):
         self.assertEqual(m.get('state'), u'deleted')
         self.assertEqual(m.get('foo'), u'baz')
 
-    def test_botgroup(self):
```

Comment:
Good question, it was mostly about the fact, that we are testing test method. And this test to pass requires different implementation of ``start_job`` (because default ``project.jobq`` starts job no matter was botgroup defined or not). So, current simplified version of ``start_job``, which uses only project's jobq, wouldn't be enough and we need to go back to our old good version of ``client.start_job`` with checking did we receive projectid or not (only maybe without auth argument). From that perspective seems it's 'too much' for testing test method. But I'm ok with keeping it too, of course :) So, what do you think?